packages/ckeditor5-package-generator/lib/templates/ts/typings/ckeditor5-inspector.d.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license Copyright (c) 2022-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
```

Comment:
Is this header needed? Other files from the templates does not contain license header.

Reply:
This got here by mistake, good catch.